packages/frontend/src/components/BottomBar.tsx

```diff
@@ -0,0 +1,49 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faHouse,
+  faMagnifyingGlass,
+  faPerson,
+  faTrophy,
+} from "@fortawesome/free-solid-svg-icons";
```

Comment:
Add a newline here, the linter says.
```suggestion
} from "@fortawesome/free-solid-svg-icons";

```